Tests/CoffeeAddictsTests/CoffeeAddictsTests.swift

```diff
@@ -0,0 +1,113 @@
+import XCTest
+import class Foundation.Bundle
+@testable import CoffeeAddictsLibrary
+
+final class CoffeeAddictsTests: XCTestCase {
+    func testValidClosestLocationToUserLocation() {
+        let userLocation = CLLocation(latitude: 42.21, longitude: 22.21)
+        let coffeeShop = CoffeeShop(name: "Mock Coffee Shop", coords: userLocation)
+
+        let closestLocation = CoffeeShopsLocationHelper.closestLocation(locations: [coffeeShop], closestToLocation: userLocation)
+
+        XCTAssert(closestLocation.count == 1)
+    }
+
+    func testInvalidLocation() {
+        let location = CLLocation(latitude: -91.2, longitude: 182.2)
+        let result = CoffeeShopsLocationHelper.locationIsValid(location: location)
+        XCTAssertFalse(result)
+    }
+    
+    func testInvalidCSVFile() {
+//        let bundle = Bundle(for: type(of: self))
```

Comment:
Did you forget to remove the comments?

Reply:
Sort of, left them on purpose in case i find a good solution instead to read a file from bundle in an executable Swift Package. Currently the approach is buggy for Swift Packages that don't have a project file. 

I removed them now.